ntbs-service/Models/Entities/PreviousTbService.cs

```diff
@@ -1,10 +1,13 @@
 ﻿using System;
+using System.ComponentModel.DataAnnotations;
 using EFAuditer;
 using Microsoft.EntityFrameworkCore;
 
 namespace ntbs_service.Models.Entities
 {
     [Owned]
+    [Display(Name = "Previous TB Service")]
```

Comment:
When I fixed the bug with SystemEdited, I discovered another bug with the notification overview page from NTBS-2986, which this change fixes

Reply:
Good spot, guessing the audits were trying to get the display name to tell the user what had been changed/added?

Reply:
Exactly!